src/features/targeting/index.js

```diff
@@ -1,4 +1,10 @@
 // @flow
-export * from './contexts';
+export * from './constants';
 
 export * from './types';
+
```

Comment:
Likewise, empty lines necessary?

Reply:
remove empty lines as there is no real need for separation